### DISCUSSION TITLE
Welcome to miller Discussions!

### DISCUSSION THREAD

**Discussion by johnkerl** [johnkerl/miller#404] [General]
## 👋 Welcome!
Following on https://github.com/johnkerl/miller/issues/176, we’re trialing GitHub Discussions as a place to connect with other members of Miller community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * As always, feel free to submit bug reports and/or feature requests at https://github.com/johnkerl/miller/issues
  * Welcome others and are open-minded. Remember that this is a community we build together 💪

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by derekmahar** [comment]
I've used Miller to transform cryptocurrency exchange trading history reports to an internal normal form and to calculate capital gains according to Canadian tax rules.

Over the past few years, I've opened a handful of [issues](https://github.com/johnkerl/miller/issues/created_by/derekmahar) that I hope @johnkerl will address sooner than later.  (Hint, hint, nudge, nudge!)

**Reply by johnkerl** [reply]
Thanks! The [Go port](https://github.com/johnkerl/miller/labels/go-port) is where I'm putting all available time -- this will resolve a host of issues, including some of yours, and will then unblock more feature-development.

The DSL work is largely done and most of the verbs are ported. Regression-testing I'm doing as I go. Doc-work is a bit lagging behind code-work ...

**Reply by derekmahar** [reply]
I didn't know that you were porting Miller to Go.  Why did you choose Go?

**Reply by johnkerl** [reply]
Lots of reasons here -- https://github.com/johnkerl/miller/blob/master/go/README.md :)

**Comment by aborruso** [comment]
Miller and especially @johnkerl change my daily working life.

Miller is an incredible both powerful and simply tool and John is a great developer and a nice person, available to listen and always ready to make new proposals.

For these reasons I have opened a lot of [issues](https://github.com/johnkerl/miller/issues?q=is%3Aissue+author%3Aaborruso+) and I'm very proud and happy that some of these have become new features. 

It seems absurd to me that Miller is little known and I do my best to make it known in places like [Stack Exchange](https://github.com/johnkerl/miller/issues/212#issuecomment-451508839).

Once again thank you very much John

**Reply by NikosAlexandris** [reply]
Great words that deserve to be said.-

Miller deserves to be more popular.

**Comment by trantor** [comment]
I might as well add my intro as well.
I am Fulvio and I am an Italian IT system engineer.
I love to death both using the *nix terminal and using data tools to manipulate, transform and extract meaning from data. Vim, regex lover and perler too, unsurprisingly. Compulsively starring anything on Github which is data/cli oriented.
I've used miller to do pretty much anything I could think of with tabular data, sometimes even when any sensible person would have switched to a general-purpose language.
I too enormously appreciate the tool, especially in combination with others such as `jq`, just as much as I appreciate @johnkerl 's attention to its users and their proposals/issues. I've been bug-finding and nit-picking about the documentation, with several issues opened a few PRs here and there.
Recently I've started to train colleagues `@work` and I've spreading the word about miller with quite the response. Probably going to start some blog post and/or public tutorial about it in the future as well.
